modules/earthengine-layers/package.json

```diff
@@ -22,7 +22,7 @@
     "@math.gl/web-mercator": "3.1.2"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^11.0.2",
+    "@rollup/plugin-commonjs": "^15.0.0",
```

Comment:
Should we also bump earthengine api module to latest?

Reply:
If so, maybe better as separate PR.

Reply:
Yeah probably good for a different PR